SPECS/openmpi/openmpi.spec

```diff
@@ -0,0 +1,1139 @@
+# Optional name suffix to use...we leave it off when compiling with gcc, but
+# for other compiled versions to install side by side, it will need a
+# suffix in order to keep the names from conflicting.
+#global _cc_name_suffix -gcc
+
+%global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
+%ifarch aarch64 ppc64le x86_64
+%bcond_without ucx
+%else
+%bcond_with ucx
+%endif
+# ARM 32-bit is not supported by rdma
+# https://bugzilla.redhat.com/show_bug.cgi?id=1780584
+%ifarch %{arm}
+%bcond_with rdma
+%else
+%bcond_without rdma
+%endif
+# No more Java on i686
+%ifarch %{java_arches}
+%bcond_without java
+%else
+%bcond_with java
+%endif
+# Private openmpi libraries
+%global __provides_exclude_from %{_libdir}/openmpi/lib/(lib(mca|ompi|open-(pal|rte|trace))|openmpi/).*.so
+%global __requires_exclude lib(mca|ompi|open-(pal|rte|trace)|vt).*
+# Run autogen - needed for some patches
+%bcond_with autogen
```

Comment:
Since it's unconditionally disabled, let's just remove the macro and the spec sections, which are now disabled:
```suggestion
```